src/modules/85-cv/components/Notifications/components/NotificationRuleRow/NotificationRuleRow.tsx

```diff
@@ -166,28 +212,28 @@ export default function NotificationRuleRow({
   index
 }: NotificationRuleRowProps): JSX.Element {
   const { getString } = useStrings()
-  const { changeType, id, condition, threshold } = notificationRule
+  const SRM_CODE_ERROR_NOTIFICATIONS = useFeatureFlag(FeatureFlag.SRM_CODE_ERROR_NOTIFICATIONS)
+
+  const actualConditionOptions = SRM_CODE_ERROR_NOTIFICATIONS
+    ? conditionOptions
+    : conditionOptions.filter(el => el.value != 'CodeErrors')
```

Comment:
Can you use the enum for this?
```suggestion
    : conditionOptions.filter(el => el.value != Condition.CODE_ERRORS)
```